### PR TITLE
@joeyAghion => [CCPA] Initial start for CCPA modals

### PR DESCRIPTION
**Pull request by mzikherman**
This is super rough, but just starts it off, based on https://github.com/artsy/gravity/pull/12707

A mutation for that needs to be added, but you get the idea!

search

### CODE REVIEW

File: src/Components/v2/CCPARequest.tsx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    <SuccessScreen
+      text={feedbackText}
+      email={userEmail || email}
+      name={userName || name}
+    />
+  ) : !user ? (
+    <LoggedOutContents
+      setName={setName.bind(this)}
+      setEmail={setEmail.bind(this)}
+      setText={setFeedbackText.bind(this)}
```
Comment (joeyAghion) [inline src/Components/v2/CCPARequest.tsx]: Does this take care of proper escaping? I always get suspicious when we re-render the user's input back to them. If this isn't part of the design's confirmation screen, I'd leave it out and avoid issues of truncation, wrapping, and escaping.

Comment (mzikherman) [reply src/Components/v2/CCPARequest.tsx]: Well, the `bind` is definitely needed, as these are callbacks to set a local state to represent the user's input. The callbacks just...set that info. There's no escaping here. There is a temporary 'Success' contents in this file, just as a placeholder to get the PR up (there was no mutation ready at the time, etc.).

Happy to pair if you want to understand these callbacks better!

**Comment by mzikherman** [comment]
Merging this in.

**Comment by artsyit** [comment]
:rocket: PR was released in v24.0.8 :rocket:

**Pull request merged**